yarn.lock

```diff
@@ -3,15 +3,15 @@
 
 
 "@apollo/client@^3.0.2":
-  version "3.0.2"
-  resolved "https://registry.yarnpkg.com/@apollo/client/-/client-3.0.2.tgz#fadb2b39a0e32950baaef2566442cb3f6de74a52"
-  integrity sha512-4ighan5Anlj4tK/tdUHs4Mi1njqXZ7AxRCVolz/H702DjPphAJfm+FRkIadPTmwz+OLO+d+tX+6V1VBshf02rg==
+  version "3.1.3"
```

Comment:
Could you please `git checkout master yarn.lock` and reinstall dependencies? I'm pretty sure there shouldn't be that many changes and we wouldn't want to sneak in untested dependencies upgrades, wouldn't we? 😅